apps/pipelines/models.py

```diff
@@ -62,7 +62,7 @@ class Pipeline(BaseTeamModel, VersionsMixin):
     objects = PipelineManager()
 
     class Meta:
-        ordering = ["-created_at"]
+        ordering = ["name"]
```

Comment:
since you're overriding this in the view you could leave it as is to avoid having to create a migration: